src/plugins/plugin-build-script.ts

```diff
@@ -0,0 +1,43 @@
+import execa from 'execa';
+import npmRunPath from 'npm-run-path';
+const cwd = process.cwd();
+
+export function buildScriptPlugin({
+  input,
+  output,
+  cmd,
+}: {
+  input: string[];
+  output: string[];
+  cmd: string;
+}) {
+  if (output.length !== 1) {
```

Comment:
Wouldn't this fail if it has 2 outputs?

Reply:
Yea, the thinking here is that a build script (as in, a bash command piping stdin -> stdout) would never be able to support more than a single file output. You could never represent Svelte compiling both a JS & CSS file output at the same time, for example.

This brings up a good point that "output" might come off as an  "either or" type of thing, when it's meant to mean "this file may output multiple file types at the same time".